.github/workflows/ocsp-basic-test.yml

```diff
@@ -246,6 +246,104 @@ jobs:
           echo good > expected
           diff expected actual
 
+      - name: Check OCSP responder with non existing cert
```

Comment:
If I understood correctly this means that if a cert doesn't exist in the CA, the OCSP subsystem wil respond with a "good" status, but the CA's built-in OCSP responder will respond with an "unknown" status. Is this correct? Is it possible to document somewhere (e.g. in wiki) the expected value for each case?

Reply:
Yes, this is correct. The OCSP has only the list of revoked certificate and if it is not revoked it has no information. For the original specification `Good` is the best answer in this case. However, in the update specification (rfc6960) they accept the original behaviour but if possible the status should be revoked or unknown.
I have created [this page](https://github.com/dogtagpki/pki/wiki/OCSP-status-responses) and linked to OCSPClient page.

Reply:
@fmarco76 Thanks!
@ladycfu @06shalini FYI